cql2pgjson/src/test/resources/templates/db_scripts/subquery.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
Can you rename this file from subquery.json to foreignKeyMultiTable.json to be self-explanatory and to better match the other .json file names?

Reply:
leaving as is.